src/Service/EmailNotificationService.php

```diff
@@ -61,14 +61,16 @@ public function sendEmails()
                         'count' => count($participants)
                     ]);
                 } else {
-                    $message = new Message($this->env, $this->loggerService, $this->twig, $this->params);
-                    $message
-                        ->setTo($result['emails'])
-                        ->render($this->render, [
-                            $this->level => $result['id'],
-                            'participants' => $participants
-                        ])
-                        ->send();
+                    if ($this->env->isStaging() || $this->env->isProd()) {
```

Comment:
We should should send emails in stable and prod and disable in dev and staging env.